### PR TITLE
Add missing license of swck

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

**Review by fgksgf — APPROVED** [review]

File: dist/LICENSE [hunk 1]
```diff
@@ -211,6 +211,7 @@ The text of each license is the standard Apache 2.0 license.
 	mum4k (termdash) 0.10.0: https://github.com/mum4k/termdash Apache 2.0
 	yaml.v2 2.2.4: https://gopkg.in/yaml.v2 Apache 2.0
 	Google (grpc-go): https://github.com/grpc/grpc-go Apache 2.0
+	skywalking-swck v0.0.0-20210107023854-d15ef19f8317: https://github.com/apache/skywalking-swck Apache 2.0
```
Comment (wu-sheng) [inline dist/LICENSE:214]: SWCK has its dependencies two. Will CLI use it or not?

**Comment by wu-sheng** [comment]
Let's keep this, we should wait for the next SWCK release(0.2.0) and update the dependency.

**Comment by fgksgf** [comment]
It seems this PR can continue.

File: dist/LICENSE [hunk 2]
```diff
@@ -211,6 +211,9 @@ The text of each license is the standard Apache 2.0 license.
...
```
Comment (wu-sheng) [inline dist/LICENSE:214]: Please correct this?

File: go.mod [hunk 3]
```diff
@@ -5,8 +5,7 @@ go 1.13
 replace skywalking/network v1.0.0 => ./gen-codes/skywalking/network
 
 require (
-	github.com/apache/skywalking-swck v0.0.0-20210107023854-d15ef19f8317
-	github.com/ghodss/yaml v1.0.0
+	github.com/apache/skywalking-swck v0.0.0-20210120061318-74f6b6b88c58
```
Comment (wu-sheng) [inline go.mod:8]: Please update this to 0.2.0

File: dist/LICENSE (hunk 2 above)
Comment (wu-sheng) [inline dist/LICENSE:214]: And then, you forget to update this...

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**